docs/user-guide/configure-tls.md

```diff
@@ -108,50 +107,54 @@ be specified as separate backends.
 
 ### Backend Application Attributes:
 
-  - trustAllCerts - Whether (true) or not (false) to authenticate server side.
+
+  - trustAllCerts - Turns off (true) or on (true) client-side TLS authentication. When `trustAllCerts` is false,
+  origin servers need to provide a trusted certificate or the connection will be rejected.
 
   - *sslProvider* - Specifies a Java service provider implementation for the TLS protocol.
     Supported values are `JDK` and `OPENSSL`. Note that `OPENSSL` is a platform dependent
     (Linux/Mac/Windows) implementation and you must use an appropriate Styx build for your
     deployment target.
 
   - *addlCerts* - A list of additional certificates that can be specified
-    in addition to the certificates available in Trust Store. When Styx starts
-    up it creates an in-memory keystore and loads these certificates there.
+    in addition to the certificates available in the Trust Store. When Styx starts
+    up, it creates an in-memory keystore and loads these certificates there.
     Each specified certificate entry must have these two attributes:
 
-      - *alias* - A symbolic name for the certificate. Used as an alias for keystore.
+      - *alias* - A symbolic name for the certificate. Used as an alias in the keystore.
 
       - *path*  - A path to the certificate file.
 
-  - *trustStorePath* - Styx can load the relevant secure material from Java truststore.
-    This attribute specifies a path to the keystore file containing relevant trust material.
+  - *trustStorePath* - Styx can load the list of trusted certificates from a Java JKS keystore.
+    This attribute specifies a path to the truststore file containing the trusted certificates.
```

Comment:
By convention this file where trust material is stored is called *truststore* which technically is just a java keystore file. In this regard the description feels bit more confusing than it was before.